lib/css_parser/rule_set.rb

```diff
@@ -27,16 +27,17 @@ class RuleSet
     class Declarations
       class Value
         attr_reader :value
-        attr_accessor :important, :order
+        attr_accessor :important
 
-        def initialize(value, important: nil, order: 0)
+        def initialize(value, important: nil)
           self.value = value
-          @important = important.nil? ? !value.match(CssParser::IMPORTANT_IN_PROPERTY_RX).nil? : important
-          @order = order
+          @important = important unless important.nil?
```

Comment:
it's nil either way
```suggestion
          @important = important
```

Reply:
Not anymore, we're assigning it in `value=`.